xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1319,6 +1319,7 @@ watchlist.event.create=On {0}, the page has been created by {1}
 watchlist.event.delete=On {0}, the page has been deleted by {1}
 watchlist.event.update=On {0}, the page has been modified by {1}
 watchlist.event.update.multiple=Between {0} and {1}, the page has been modified {2} times, by {3} user{3,choice,0#s|1#|2#s}: {4}
+watchlist.notifications=Notifications
```

Comment:
`watchlist.notification.tooltip=Notifications` seems to be more in line with the translations' conventions described in https://dev.xwiki.org/xwiki/bin/view/Community/L10N%20Conventions.